Material.Blazor/Components/DatePicker/InternalDatePickerPanel.razor.cs

```diff
@@ -213,6 +213,7 @@ private void OnNextMonthClick()
         /// <inheritdoc/>
         protected override async Task OnAfterRenderAsync(bool firstRender) // TODO: this prevents us from marking OnAfterRenderAsync in InputComponent as sealed. Consider alternatives!
         {
+            await base.OnAfterRenderAsync(firstRender);
```

Comment:
We perform relevant work in base.OnAfterRenderAsync, so we can't simply omit that.